tests/unit/account_info_query_test.py

```
@@ -19,43 +19,49 @@
     query = AccountInfoQuery(account_id)
     assert query.account_id == account_id
 
+
 def test_execute_fails_with_missing_account_id(mock_client):
     """Test request creation with missing Account ID."""
     query = AccountInfoQuery()
-    
-    with pytest.raises(ValueError, match="Account ID must be set before making the request."):
+
+    with pytest.raises(
+        ValueError, match="Account ID must be set before making the request."
```

Ruff (0.14.13)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)